docs/content/release/0.22.0-notes.rst

```diff
@@ -0,0 +1,216 @@
+
+June 26th, 2024
+
+.. contents:: Highlights
+    :depth: 3
+
+Updates
+=======
+
+New features
+------------
+
+Now we can use a faster and more memory efficient implementation of the
+magnetic simulation
+:class:`~simpeg.potential_fields.magnetics.Simulation3DIntegral`, making use of
+Choclo and Numba. To make use of this functionality you will need to `install
+Choclo <https://www.fatiando.org/choclo/latest/install.html>`__ in addition to
+``simpeg``. This new implementation is able to forward model TMI and all
+magnetic field components with susceptibility (scalar) or effective
+susceptibility (vector) models.
+
+Documentation
+-------------
+
+Since v0.22.0 we host the documentation pages for `latest
+<https://docs.simpeg.xyz/latest>`__ and older versions of SimPEG, along with
+the pages for the `latest development version <https://docs.simpeg.xyz/dev>`__.
+The docs now include a version switcher in the top bar that will allow users to
+navigate between the pages for each different version.
+
+Several improvements have been made to the documentation pages.
+
+In the `Getting Started
+<https://docs.simpeg.xyz/latest/content/getting_started/index.html>`__ guide we
+stopped recommending ``mamba`` as the best package manager for installing
+SimPEG after ``conda`` started using ``libmamba`` under the hood, achieving the
+same performance as ``mamba``. We also included instructions so our
+contributors can easily update their local environment, which is specially
+useful after we update the minimum version of the development dependencies
+(such as autoformatters and linters).
+
+The documentation for Frequency-Domain (FDEM) and Time-Domain (TDEM) EM fields
+and 3D simulations got greatly extended, with more details on the parameters
+for each method, along with more mathematical background for their
+implementations.
+
+We fixed the links to the source code for each class, function and method:
+they now point to the corresponding version in the GitHub repository.
+
+Lastly, we applied general fixes to typos, the contour colors for one of the
+gravity examples, and to the broken links to the source code for each class,
+function and method. These links now point to their corresponding version in
+the GitHub repository.
+
+Bugfixes
+--------
+
+This release comes with a few bug fixes. We solved an issue with the distance
+calculation in a utility function that converts 3D DC-IP survey locations to 2D
+lines. We fixed a bug on how the arguments passed to the directives that
+estimate the beta parameter where being passed to the parent classes. We
+updated the code in :class:`~simpeg.utils.pgi_utils.GaussianMixtureWithPrior`
+to make it compatible with the latest versions of scikit-learn. And we now make
+sure that the queues are joined when the meta simulation is joined.
+
+Breaking changes
+----------------
+
+SimPEG has been renamed
+^^^^^^^^^^^^^^^^^^^^^^^
+
+SimPEG v0.22.0 comes with a major change that everyone will experience! We are
```

Comment:
I think this section should come first as it is the one most people will see in their everyday use.